api/v1alpha2/cluster_types.go

```diff
@@ -138,7 +142,7 @@ type APIEndpoint struct {
 	Port int `json:"port"`
 }
 
-/// [APIEndpoint]
+// ANCHOR_END: APIEndpoint
```

Comment:
Are these Anchors currently used anywhere and any particular reason why we wouldn't reference an anchor for the `Cluster` type?

Reply:
Not right now, we need to add all the docs for each controller and type. These can be used to reference code